lib/utils/bpr.js

```diff
@@ -11,16 +16,34 @@ function _getConditions(node) {
   _getCallee(node.test, callees);
 
   if (node.test.left) {
+    // If the global reference is part of the conditional and is on the left side (i.e. it's before the
+    // isBrowser check), return an empty array of callees to make isEnvironmentBrowser/isBrowser check false.
```

Comment:
Can we refactor the conditional to remove the need for these comments? That may mean creating intermediate variables/functions to make the check more explicit. 

I'm much prefer this vs. comments as comments become out of date, but code doesn't.

Reply:
Done